wagtail/contrib/search_promotions/tests.py

```diff
@@ -848,3 +875,29 @@ def test_query_popularity(self):
         self.assertEqual(popular_queries[0], Query.get("unpopular query"))
         self.assertEqual(popular_queries[1], Query.get("popular query"))
         self.assertEqual(popular_queries[2], Query.get("little popular query"))
+
+
+class TestSearchPromotionPermission(WagtailTestUtils, TestCase):
+    def setUp(self):
+        self.user = self.login()
+
+    def get(self, params={}):
+        return self.client.get(reverse("wagtailsearchpromotions:index"), params)
+
+    def test_simple(self):
+        response = self.get()
+        self.assertEqual(response.status_code, 200)
+
+    def test_get_with_no_permission(self):
+        self.user.is_superuser = False
+        self.user.save()
+        self.user.user_permissions.add(
+            Permission.objects.get(
+                content_type__app_label="wagtailadmin", codename="access_admin"
+            )
+        )
+        # No Permissions created
```

Comment:
I think it'd be great to have another test case with a user that only has a minimal set of permissions, e.g. just the `edit` permission for the `Query` model to prove that it works with a non-superuser.